doc/source/release/1.11.0-notes.rst

```diff
@@ -93,6 +230,31 @@ Other
 Deprecated features
 *******************
```

Comment:
@stefanv could you suggest an entry for your recent sparse deprecation

Reply:
Sure! It's holiday in the US Monday, so if this can happen on Tuesday that'd be great.

Reply:
CJ drafted some deprecations in, and I added a few more locally based on the sprint notes/PRs, that I'll push shortly. I'll welcome refinements even after I branch, so a PR can be opened later.